packages/libs/map/types.ts

```diff
@@ -241,16 +298,19 @@ export type CompanyItem = BaseItem &
 export type Cutscene = BaseItem &
   Readonly<{
     type: ItemType.Cutscene;
+    dlcGuard: number;
     flags: number;
     tags: readonly string[];
     nodeUid: bigint;
+    actionStringParams: readonly string[];
   }>;
 
 export type Trigger = BaseItem &
   Readonly<{
     type: ItemType.Trigger;
     dlcGuard: number;
-    actionTokens: readonly string[];
+    // [action token, params] tuples
+    actions: [string, readonly string[]][];
```

Comment:
I originally wrote this as `Map<string, string[]>`. But then `JSON.serialize`ing Triggers resulted in empty objects being serialized 😕 

I don't think we're at the point where we need special serialize/deserialize routines yet... but maybe in the future.